tests/internal/fuzzers/flb_json_fuzzer.cpp

```diff
@@ -0,0 +1,20 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+extern "C" {
```

Comment:
why the fuzzer is a .cpp and not a .c ? (we prefer c)

Reply:
Changed to C now!